server/environment.go

```diff
@@ -109,27 +114,29 @@ func (s *Server) registerEnvironmentRoutes(g *echo.Group) {
 		}
 
 		for _, item := range patchList {
-			environmentPatch, _ := item.(*api.EnvironmentPatch)
-			environmentPatch.UpdaterID = c.Get(getPrincipalIDContextKey()).(int)
-			_, err = s.EnvironmentService.PatchEnvironment(ctx, environmentPatch)
-			if err != nil {
+			envPatch, _ := item.(*api.EnvironmentPatch)
```

Comment:
nit: would you do an error here here though not introduced by this change? Just in case we forget about it.

Reply:
Done.